Gems/Atom/Tools/ShaderManagementConsole/Code/Source/Window/ShaderManagementConsoleWindow.cpp

```diff
@@ -37,6 +38,62 @@ namespace ShaderManagementConsole
         Base::OnDocumentOpened(documentId);
         m_documentInspector->SetDocumentId(documentId);
     }
+
+    void ShaderManagementConsoleWindow::UpdateMenus(QMenuBar* menuBar)
+    {
+        Base::UpdateMenus(menuBar);
+        m_actionSaveAsChild->setVisible(false);
+    }
+
+    AZStd::string ShaderManagementConsoleWindow::GetSaveDocumentParams(const AZStd::string& initialPath) const
+    {
+        // Get shader file path
+        AZStd::string shaderFullPath;
+        AZ::RPI::ShaderVariantListSourceData shaderVariantList = {};
+        ShaderManagementConsoleDocumentRequestBus::EventResult(
+            shaderVariantList,
+            GetCurrentDocumentId(),
+            &ShaderManagementConsoleDocumentRequestBus::Events::GetShaderVariantListSourceData);
+        shaderFullPath = AZ::RPI::AssetUtils::ResolvePathReference(initialPath, shaderVariantList.m_shaderFilePath);
+        
+        QMessageBox msgBox;
+        msgBox.setText("Where do you want to save the list?");
+        QPushButton* projectBtn = msgBox.addButton(QObject::tr("Save to project"), QMessageBox::YesRole);
+        QPushButton* engineBtn = msgBox.addButton(QObject::tr("Save to engine"), QMessageBox::NoRole);
+        msgBox.exec();
+
+        AZStd::string result;
```

Comment:
This result type can be an AZ::IO::Path
```suggestion
        AZ::IO::Path result;
```